pkg/collector/corechecks/snmp/common/utils.go

```diff
@@ -42,3 +43,31 @@ func CopyStrings(tags []string) []string {
 func GetAgentVersionTag() string {
 	return "agent_version:" + version.AgentVersion
 }
+
+// NormalizeHost applies a liberal policy on host names.
+func NormalizeHost(host string) string {
+	var buf bytes.Buffer
+
+	// hosts longer than 253 characters are illegal
+	if len(host) > 253 {
+		return ""
```

Comment:
Should we also force a maximum size on `namespace` to prevent falling in that case? Feeling like this can become very difficult to debug why the hostname would be empty otherwise.

Reply:
I could edit the normalization function to normalize namespace instead of hostname. My attempt is to keep normalization code as close as possible to backend code.